hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/functional/TestCOWDataSourceStorage.scala

```diff
@@ -232,7 +235,7 @@ class TestCOWDataSourceStorage extends SparkClientFunctionalTestHarness {
     val timeTravelDF = spark.read.format("org.apache.hudi")
       .option(DataSourceReadOptions.QUERY_TYPE.key, DataSourceReadOptions.QUERY_TYPE_INCREMENTAL_OPT_VAL)
       .option(DataSourceReadOptions.BEGIN_INSTANTTIME.key, "000")
-      .option(DataSourceReadOptions.END_INSTANTTIME.key, firstCommit)
+      .option(DataSourceReadOptions.END_INSTANTTIME.key, completionTime1)
       .load(basePath)
```

Comment:
Should this be from `completionTime1` to `completionTime2`?

Reply:
Same as above, this range is supposed to be `(completionTime1, completionTime1]`. And any timestamp that's before completionTime1 would suffice as the start time